lib/omniship/newgistics.rb

```diff
@@ -0,0 +1,29 @@
+require 'omniship/newgistics/track'
+
+module Omniship
+  module Newgistics
+    LABEL = "Newgistics"
+    TRACKING_REGEX = /TODO/
```

Comment:
we don't really have to wait for real data since we pretty much know what the tracking numbers will look like (for us anyway)